content/sensu-enterprise/3.0/integrations/opsgenie.md

```diff
@@ -49,6 +64,8 @@ event handler (integration).
 The following attributes are configured within the `{"opsgenie": {} }`
 [configuration scope][3].
 
+_NOTE: Some attributes are only available for certain tiers of OpsGenie._
```

Comment:
This reads a little funny to me. Perhaps `Some attributes are only available for certain OpsGenie subscription tiers.` ?